www/src/pages/table/datatable.mdx

```diff
@@ -8,7 +8,8 @@ devStatus: 'In progress'
 
 import { StaticQuery, graphql } from 'gatsby';
 import get from 'lodash/get';
-import { DataTable, TextFilter } from '~paragon-react';
+import { useMemo } from 'React';
```

Comment:
I think this is causing the doc site build to fail

Reply:
Yessss and thankfully it was something I thought I needed that I don't actually need.